docs/examples/workflows/script_annotations_outputs.md

```diff
@@ -88,17 +88,17 @@
           command:
           - python
           env:
-          - name: hera__script_annotations
+          - name: HERA__SCRIPT_ANNOTATIONS
             value: ''
-          - name: hera__outputs_directory
+          - name: HERA__OUTPUTS_DIRECTORY
             value: user/chosen/outputs
           image: python:3.8
           source: '{{inputs.parameters}}'
           volumeMounts:
           - mountPath: user/chosen/outputs
-            name: hera__outputs_directory
+            name: hera--outputs-directory
```

Comment:
@dejamiko can we get rid of the double hyphens here?

Reply:
Sure!